docs/polyglot/polyglot-bindings.md

```diff
@@ -161,22 +161,22 @@ the language-specific documentation for details.
 ## Embedded Syntax
 
 The term "Embedded Syntax" is our terminology for the ability to use foreign
-language syntaxes from directly inside `.enso` files. This system builds upon
+language syntaxes directly from inside `.enso` files. This system builds upon
 the more generic mechanisms used by the [polyglot FFI](#the-polyglot-ffi) to
 provide a truly seamless user experience.
 
 ### Embedded Syntax Usage (Syntax)
 
 A polyglot block is introduced as follows:
 
-- The `polyglot` keyword starts a block.
+- The `foreign` keyword starts a block.
 - This must be followed by a language identifier (e.g. `java`).
 - After the language identifier, the remaining syntax behaves like it is an Enso
   function definition until the `=`.
 - After the `=`, the user may write their foreign code.
```

Comment:
There is no mention of the `"""` quote that seems to also be needed for foreign code. It is shown in the example, but for completeness I think we should also mention it here.